src/inclusive_map.rs

```diff
@@ -48,6 +49,45 @@ where
     }
 }
 
+impl<K, V> PartialEq for RangeInclusiveMap<K, V, K>
```

Comment:
These impls should have parameters like this:

```
impl<K, V, StepFnsT> RangeInclusiveMap<K, V, StepFnsT>
where
    K: Ord + Clone,
    V: Eq + Clone,
    StepFnsT: StepFns<K>,
{
```

I.e. StepFnsT shouldn't be hard-coded to be the same as K.